planner/core/logical_plan_builder.go

```diff
@@ -3148,7 +3148,7 @@ func (b *PlanBuilder) buildWindowFunctions(p LogicalPlan, groupedFuncs map[*ast.
 				return nil, nil, err
 			}
 			if desc == nil {
-				return nil, nil, ErrWrongArguments.GenWithStackByArgs(windowFunc.F)
+				return nil, nil, ErrWrongArguments.GenWithStackByArgs(strings.ToLower(windowFunc.F))
```

Comment:
Does `ErrWindowInvalidWindowFuncUse` need to lower the function name?

Reply:
`ErrWindowInvalidWindowFuncUse` is not mentioned in #11009.
I think it's another issue.

Reply:
#11508